docker-compose.yml

```
@@ -54,14 +41,14 @@ services:
       - "1234:1234"
       - "1235:1235"
       - "3000:3000"
+      - "3035:3035"
     depends_on:
       - db
       - redis
       - docker-host
-      - webpack
     environment:
       - RAILS_DEV_DB_PASS=password
-      - WEBPACKER_DEV_SERVER_HOST=webpack
+      - WEBPACKER_DEV_SERVER_HOST=web
     stdin_open: true
```

I think this might be useful to help your shell do job control
```suggestion
    init: true
    stdin_open: true
```
ref https://docs.docker.com/compose/compose-file/05-services/#init

---

Already there on line 32!